src/pages/home/ErrorPage.tsx

```
@@ -0,0 +1,30 @@
+import React from 'react';
+import Error from '../../assets/images/error.svg';
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)